### PR TITLE
Reduce ChannelTimeout to 50 in Granite

### PR DESCRIPTION
**Pull request by Inphi**
Description

Reduce the ChannelTimeout in Granite hard fork from 300 to 50.

### CODE REVIEW

File: op-chain-ops/genesis/genesis.go [hunk 1]
```diff
@@ -67,7 +67,8 @@ func NewL2Genesis(config *DeployConfig, block *types.Block) (*core.Genesis, erro
 		CancunTime:                    config.EcotoneTime(block.Time()),
 		EcotoneTime:                   config.EcotoneTime(block.Time()),
 		FjordTime:                     config.FjordTime(block.Time()),
-		InteropTime:                   config.InteropTime(block.Time()),
+		// TODO: GraniteTime:                   config.GraniteTime(block.Time()),
```
Comment (Inphi) [inline op-chain-ops/genesis/genesis.go]: depends on types introduced in https://github.com/ethereum-optimism/op-geth/pull/353

**Comment by ajsutton** [comment]
@Inphi I've pulled in the op-geth build with the latest changes and resolved the TODOs, plus fixed the test failures so I think this should be good to go now (assuming the tests pass with the updated op-geth).

**Review by ajsutton — APPROVED** [review]

**Review by mbaxter — APPROVED** [review]
Contract changes LGTM

**Comment by mslipper** [comment]
Infra changes LGTM.

**Review by mslipper — APPROVED** [review]

File: op-node/rollup/sequencing/sequencer.go [hunk 2]
```diff
@@ -526,6 +526,11 @@ func (d *Sequencer) startBuildingBlock() {
 		d.log.Info("Sequencing Fjord upgrade block")
 	}
 
+	// For the Fjord activation block we shouldn't include any sequencer transactions.
```
Comment (mdehoog) [inline op-node/rollup/sequencing/sequencer.go:529]: @Inphi comment is outdated... did you want to set `attrs.NoTxPool = true` or is this just for logging purposes?

Comment (ajsutton) [reply op-node/rollup/sequencing/sequencer.go:529]: We should not set NoTxPool for the granite activation block - there's no upgrade transactions so no reason to exclude L2 transactions. We may have lost the comment update in a cherry-pick somewhere along the line (or I just dreamt that I fixed it at one point...)

**Pull request merged**